example/hybrid-mbr.nix

```diff
@@ -0,0 +1,62 @@
+{ disks ? [ "/dev/sda" ], ... }:
```

Comment:
Do you have a suggestion how we should name `hybrid.nix` in that case since it's still a bit confusing to have both.

Reply:
As I comment here https://github.com/nix-community/disko/issues/166#issuecomment-1455683887 existent `gpt-bios-compat.nix` is enough. `hybrid.nix` or `hybrid-tmpfs-onroot.nix` looks confusing and redundant for me.

Reply:
hmm, the gpt-bios-compat can't do EFI because it has no efi or boot partition

Reply:
Yes. I would also prefer something that can support both EFI/BIOS. We have installer that works with both EFI and BIOS and makes it unnecessary for people to have to figure out what mode there machines was booted from.

Reply:
Sure, so maybe we could reorganize as:

remove `gpt-bios-compat.nix` (I'm not sure which is the use case of this without EFI partition)
remove `hybrid-tmpfs-onroot.nix` (or not, but for clarity on discussion)
`hybrid.nix` -> `efi-bios-compat.nix`

or, if we want to keep the hybrid keyword:

`hybrid_boot.nix` vs `hybrid_mbr.nix`

Also, being stricts, seems like the real name of hybrid boot using `bios_grub` flags is GPT-only protocol. As in https://wiki.archlinux.org/title/Arch_boot_process#Feature_comparison and https://repo.or.cz/syslinux.git/blob/HEAD:/doc/gpt.txt

Reply:
Hi, maybe can we get this to a conclusion?

Reply:
maybe we can call it bios-efi-boot? since it does both?